### PR TITLE
Add special vehicle details to move

### PR DESCRIPTION
**Pull request by doctorpod**
These are also cloned in re-use
Ticket: https://trello.com/c/5ESF0yZi

### CODE REVIEW

File: app/services/escort_creator.rb [hunk 1]
```diff
@@ -54,4 +58,14 @@ def deep_clone_escort
       except: EXCEPT_GRAPH
     )
   end
+
+  def build_move_with_special_vehicle_details(escort)
```
Comment (cesidio) [inline app/services/escort_creator.rb]: This is not consistent with the approach we use when cloning an existing escort. I would add the move to INCLUDE_GRAPH, and all the fields we are not interested in EXCEPT_GRAPH.

**Review by cesidio — CHANGES_REQUESTED** [review]

File: app/controllers/escorts_controller.rb [hunk 2]
```diff
@@ -12,8 +12,7 @@ def show
   end
 
   def create
-    escort = EscortCreator.call(escort_params)
-    escort.create_move(from_establishment: current_user.establishment(session))
+    escort = EscortCreator.call(escort_params, current_user.establishment(session))
```
Comment (cesidio) [inline app/controllers/escorts_controller.rb:15]: 👍 makes sense to create the move in the EscortCreator

File: app/services/escort_creator.rb (hunk 1 above)
Comment (doctorpod) [reply app/services/escort_creator.rb]: Willdo

File: spec/services/escort_creator_spec.rb [hunk 3]
```diff
@@ -146,6 +164,15 @@ def expect_offences_to_be_cloned(existent_escort, new_escort)
     expect(offences_attributes).to match_array(expected_offences_attributes)
   end
 
+  def expect_move_special_vehicle_details_to_be_cloned(existent_escort, new_escort)
+    %i[require_special_vehicle require_special_vehicle_details
```
Comment (cesidio) [inline spec/services/escort_creator_spec.rb]: To be consistent with the rest of the spec, I would add this attributes to the method:
```
except_move_attributes
```

File: spec/services/escort_creator_spec.rb [hunk 4]
```diff
@@ -146,6 +164,15 @@ def expect_offences_to_be_cloned(existent_escort, new_escort)
     expect(offences_attributes).to match_array(expected_offences_attributes)
   end
 
+  def expect_move_special_vehicle_details_to_be_cloned(existent_escort, new_escort)
```
Comment (cesidio) [inline spec/services/escort_creator_spec.rb]: Also for consistency, I would make this method similar to what we do for the cloned detainee:

```
  def expect_move_to_be_cloned(existent_escort, new_escort)
    move_attributes = existent_escort.move.attributes.except(*except_move_attributes)
    expect(new_escort.move.id).not_to eq(existent_escort.move.id)
    expect(new_escort.move).to have_attributes(move_attributes)
  end
```

**Review by cesidio — APPROVED** [review]

**Pull request merged**